<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary Counter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="9">
     <widget class="QLCDNumber" name="bit_1_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLCDNumber" name="bit_16_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="bit_2_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLCDNumber" name="bit_2_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="bit_128_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="bit_1_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="bit_32_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="bit_64_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLCDNumber" name="bit_8_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="bit_128_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="bit_16_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="bit_64_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLCDNumber" name="bit_32_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLCDNumber" name="bit_4_lcd">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="digitCount">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="bit_4_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="bit_8_button">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="binary_output_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
